### ISSUE TITLE
Resolve Moderate Dependabot Alerts

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#58688]
### Issue Description
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot?q=is%3Aopen+sort%3Amost-important

Alerts have begun to pile up in the vets-website repository. Once the high alerts have been dealt with, the moderate ones are next on the list.

---

### Tasks

- [ ] Resolve all moderate dependabot alerts

**Comment by rmessina1010** [comment]
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/122
and
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/11
resolved.

**Comment by rmessina1010** [comment]
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/64
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/53
Are the same.  They will require  on e separate ticket.
 The dependency in question  , when bumped to ^16.5.0  crashes the unit tests before they start.  Also it seems to imported only by  platform/testing ( which no-hoists it) There is also this note:
src/applications/edu-benefits/tests/pages/contactInformation.unit.spec.jsx

**Comment by rmessina1010** [comment]
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/30
is partially resolved,  BUT  is being blocked by the need to  update sass-lint  and eslint to breaking versions.
will add this to ticket list

**Comment by rmessina1010** [comment]
A new Medium level alert was added, but it is  a patch  update  for a pre existing issue. Essentially, 
https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/123 and https://github.com/department-of-veterans-affairs/vets-website/security/dependabot/110 are the same issue.
As there is no patch currently available,  a new ticket will need to research a solution for this vulnerability
